src/style.css

```diff
@@ -1,3 +1,66 @@
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+  --color: #00acdc;
+}
+
+header {
+  max-width: 1200px;
+}
+
+.nav {
+  box-sizing: border-box;
+  display: flex;
+  justify-content: space-between;
+  height: 60px;
+  background-color: #fff;
+  flex-shrink: 0;
+  padding: 0 50px;
+}
+
+.nav__list {
+  display: flex;
+  align-items: center;
+  list-style: none;
+}
+
+.nav__item {
+  font-style: normal;
+  font-weight: 500;
+  color: #fff;
```

Comment:
Add this color to CSS variable and than reuse it

Reply:
done